docs/ftldns/blockingmode.md

```diff
@@ -43,4 +45,25 @@ Queries DNS queries will be answered with an empty response (no answer section)
 ##### Disadvantage
 - Blocking page cannot be shown and whitelisting has to be performed from the dashboard or CLI
 
+## Pi-hole's `NULL` IP based blocking
```

Comment:
Why style NULL as code?

Reply:
Hmm, good question. Probably because it is the option and I wasn't sure how to make a better title. Any recommendations?

Reply:
Like the other titles, keep it in the normal style (like NXDOMAIN).